### DISCUSSION TITLE
Recommended pool now that FlexPool has announced it is closing?

### DISCUSSION THREAD

**Discussion by NostalgicGnome** [madMAx43v3r/chia-gigahorse#211] [Q&A]
Fellow FlexPool users, which pools (if any) will you be switching to now that FlexPool is shutting down? Has Max recommended any himself?

**Comment by scoville07** [comment]
I have two nodes, one is with pool.space since a long time, no issues with gigahorse. The other I switched yesterday to spacefarmers.io (Netherlands), running really great and having a very mature interfacce.

**Comment by hajes** [comment]
or solo, and do not worry about pools ;-) during GH testing. I had 350TiB C5 solo.
